extension/deployment/src/main/java/io/quarkiverse/infinispan/embedded/deployment/InfinispanEmbeddedProcessor.java

```diff
@@ -123,6 +123,10 @@ ProtobufInitializers setup(BuildProducer<ReflectiveClassBuildItem> reflectiveCla
                 .addAll(index.getAllKnownImplementors(DotName.createSimple(GeneratedSchema.class.getName())));
         List<SerializationContextInitializer> initializers = new ArrayList<>(initializerClasses.size());
         for (ClassInfo ci : initializerClasses) {
+            if (ci.isAbstract()) {
+                // don't try to instantiate an abstract class...
+                continue;
```

Comment:
oh nice catch @luigidemasi 
Would you like to add a test somewhere ?

Reply:
@karesti I added a test to cover this scenario, thanks for pointing it out. The test itself is simple, but I documented in the Javadoc what the issue was and how the fix is validated. Happy to adjust or expand if you think it would help.

Reply:
fantastic, I appreciate all the explanation there!